x-pack/test/functional/apps/machine_learning/data_visualizer/file_data_visualizer.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import path from 'path';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function({ getService }: FtrProviderContext) {
+  const ml = getService('ml');
+
+  const testDataListPositive = [
+    {
+      suiteSuffix: 'with an artificial server log',
+      filePath: path.join(__dirname, 'files_to_import', 'artificial_server_log'),
+      indexName: 'user-import_1',
+      createIndexPattern: false,
+      expected: {
+        results: {
+          title: 'artificial_server_log',
+        },
+      },
+    },
+  ];
+
+  const testDataListNegative = [
+    {
+      suiteSuffix: 'with a non-log file',
+      filePath: path.join(__dirname, 'files_to_import', 'not_a_log_file'),
+    },
+  ];
+
+  describe('file based', function() {
+    this.tags(['smoke', 'mlqa']);
```

Comment:
I noticed that ML has too many suites tagged with `smoke` label. Since we only have 1 x-pack worker running Firefox we need to select only "important" tests that gave "smoke" level coverage. Not related directly to PR, just want to point it out

Reply:
@dmlemeshko Due to flakiness on Firefox, all the ML tests have `skipFirefox` set on a outer suite level.